es6-lib/consumer.js

```diff
@@ -0,0 +1,55 @@
+import Stomp from 'stomp-client';
+import Spatial from './services/spatial';
+import logger from './util/logger';
+import ISS from './upstream/iss';
+
+function consumer(config, zookeeper, metrics) {
+  const hosts = config.amq.host
+    .replace(/stomp:\/\//g, '')
+    .split(',')
+    .map(hostAndPort => hostAndPort.split(':'));
+
+  const reconnectOpts = {
+    retries: 3,
+    delay: 500
+  };
+
+  var hostIndex = 0;
+
+  const connect = () => {
+    const [host, port] = hosts[hostIndex % hosts.length];
+
+    logger.info(`Attempting to connect to stomp://${host}:${port}`);
+    var amq = new Stomp(
+      host,
+      port,
+      config.amq.user,
+      config.amq.pass,
+      "1.0",
```

Comment:
Version 1.1 and on has a heartbeat protocol that might be handy, but it looks like this library only allows you to connect using 1.0 😞